docs/interpreter/ignite.md

```diff
@@ -89,7 +89,7 @@ After configuring Ignite interpreter, create your own notebook. Then you can bin
 
 ![Binding Interpreters](../assets/themes/zeppelin/img/docs-img/ignite-interpreter-binding.png)
 
-For more interpreter binding information see [here](http://zeppelin.apache.org/docs/manual/interpreters.html).
+For more interpreter binding information see [here](http://zeppelin.apache.org/docs/0.6.2/manual/interpreters.html).
```

Comment:
@agura I think it should be `../manual/interpreter.html` not `../manual/dynamicform.html`.